tools/sdk-ai-bots/Embeddings/settings/metadata_customized_docs.json

```diff
@@ -47,16 +47,24 @@
     "title": "AutoRest Extensions for OpenAPI 2.0",
     "url": "https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md"
   },
-  "avocado-readme.md": {
-    "title": "Avocado Readme",
-    "url": "https://github.com/Azure/avocado/blob/main/README.md"
-  },
   "control_plane_template.md": {
     "title": "Control plane pull request template",
     "url": "https://github.com/Azure/azure-rest-api-specs/blob/main/.github/PULL_REQUEST_TEMPLATE/control_plane_template.md"
   },
   "data_plane_template.md": {
     "title": "Data plane pull request template",
     "url": "https://github.com/Azure/azure-rest-api-specs/blob/main/.github/PULL_REQUEST_TEMPLATE/data_plane_template.md"
+  },
+  "directory-structure.md": {
+    "title": "azure-rest-api-specs repos directory structure",
+    "url": "https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/directory-structure.md"
+  },
+  "pr-brch-deep.md": {
+    "title": "Breaking Changes process deep dive",
+    "url": "https://eng.ms/docs/products/azure-developer-experience/design/specs-pr-guides/pr-brch-deep"
```

Comment:
@raych1 are links from `eng.ms/docs` going to work? If not, maybe from the sources, like e.g. https://dev.azure.com/azure-sdk/internal/_git/azure-sdk-docs-eng.ms?path=/docs/design/api-specs-pr/api-versions-and-branches.md

?

Reply:
@konrad-jamrozik 
You are correct. The documents under eng.ms/docs are added to the document source of the bot by default. There is no need to add them to customized docs list. The other hand, the document in the customized list must be publicly accessible. Otherwise, the pipeline cannot fetch it.